docs/releasing.md

```diff
@@ -1,39 +1,52 @@
 # Releasing
 
 To initiate a new production deployment:
+
 ```sh
 script/release vX.Y.Z
 ```
+
 See `script/release --help` for more information.
 
-> **Note:**  
-> Every production release will request an approval by the select few people before it can proceed.
+> [!NOTE]
+> Deployment workflow requires maintainer approval to run.
```

Comment:
Thank you!  I hate that we totally changed the format, so the old alerts format is a sad bother.